client/src/components/Login/index.jsx

```diff
@@ -6,8 +6,31 @@ export default class extends Component {
   state = {
     email: '',
     password: '',
+    msg:''
   };
+  handelChange = ({ target: { name, value } }) => {
+    if(name ==="email"&& value.match(/^([\w.%+-]+)@([\w-]+\.)+([\w]{2,})$/i))
+    this.setState({ [name]: value })
+    else this.setState({[name]:value})
+  }
+  handelSubmit = (e) => {
+    this.setState({msg:''})
```

Comment:
we don't need this line because the msg key is already an empty string.

Reply:
when I add a true password and email the previous result does not delete.